src/java/org/apache/poi/hssf/record/cont/ContinuableRecord.java

```
@@ -49,19 +51,26 @@ protected ContinuableRecord() {
 	 * (Note - if any {@link ContinueRecord} is required, this result includes the
 	 * size of those too)
 	 */
+	@Override
 	public final int getRecordSize() {
 		ContinuableRecordOutput out = ContinuableRecordOutput.createForCountingOnly();
 		serialize(out);
 		out.terminate();
 		return out.getTotalSize();
 	}
 
+	@Override
 	public final int serialize(int offset, byte[] data) {
-
-		LittleEndianOutput leo = new LittleEndianByteArrayOutputStream(data, offset);
-		ContinuableRecordOutput out = new ContinuableRecordOutput(leo, getSid());
-		serialize(out);
-		out.terminate();
-		return out.getTotalSize();
+		int totalSize = 0;
+		try (LittleEndianByteArrayOutputStream leo =
+				new LittleEndianByteArrayOutputStream(data, offset)) {
+			ContinuableRecordOutput out = new ContinuableRecordOutput(leo, getSid());
+			serialize(out);
+			out.terminate();
+			totalSize = out.getTotalSize();
+		} catch (IOException ioe) {
+			// should never happen in practice
```

Once more a unchecked exception would not hide a potential exception